### PR TITLE
DAOS-8984 test: Add a timeout to the RPMs test

### PR DESCRIPTION
**Pull request by brianjmurrell**
Ensure that if daos_test -m hangs, the test does not hang indefinitely.

Try to get daos_server output if daos_test -m hangs or otherwise exits
!0.

### CODE REVIEW

File: ci/rpm/test_daos_node.sh [hunk 1]
```diff
@@ -95,7 +95,8 @@ if ! module load $OPENMPI; then
     module list
     exit 1
 fi
-coproc daos_server --debug start -t 1 --recreate-superblocks
+server_start=$SECONDS
```
Comment (phender) [inline ci/rpm/test_daos_node.sh:98]: Should we be do anything with this?

Comment (brianjmurrell) [reply ci/rpm/test_daos_node.sh:98]: Removed in the next commit.

**Review by ashleypittman — APPROVED** [review]
So all this test does is start a server, then run daos_test -m?  It may be that OFI_INTERFACE isn't needed any more, it used to be a long time ago but if daos_test is using the agent then it won't be required any more.

**Review by phender — APPROVED** [review]

**Comment by brianjmurrell** [comment]
@phender, @ashleypittman Apologies for the force-push and dismissing your reviews.  Nothing has changed since your last review.  I just needed to update commit pragams to get tests of the stages that were affected by the change.

**Review by ashleypittman — APPROVED** [review]

**Review by phender — APPROVED** [review]

**Pull request merged**